Makefile

```diff
@@ -448,6 +449,10 @@ site/src/api/typesGenerated.ts: scripts/apitypings/main.go $(shell find codersdk
 	cd site
 	yarn run format:types
 
+docs/admin/prometheus.md: scripts/metricsdocgen/main.go
```

Comment:
should also have the input file as a dep

Reply:
Done